src/components/CardComponentCarousel.tsx

```diff
@@ -51,51 +47,48 @@ const CardComponentBig: React.FC<ResponseData> = (data) => {
     }, []);
     return (
         <div className="flex justify-between items-center mb-5">
-            <Images imgIndex={imgIndex} />
+            <Images imgIndex={imgIndex} imgs={imgs} items={data.items} />
         </div>
     );
 };
 
-const Images = ({ imgIndex }: { imgIndex: number }) => {
+const Images = ({ imgIndex, imgs, items }: { imgIndex: number, imgs: string[], items: Item[] }) => {
     return (
         <>
-            {imgs.map((imgSrc, idx) => {
-                return (
-                    //come back to this to fix the aspect ratio
-                    <div key={idx} className='flex flex-col items-center h-[30vh] w-1/3 rounded-xl bg-neutral-800'>
-                        <motion.div
-                            key={idx}
-                            //controls the images properties
-                            style={{
-                                backgroundImage: `url(${imgSrc})`,
-                                backgroundSize: "cover",
-                                backgroundPosition: "center",
-                            }}
-                            //this is the "focus" animation for the image
-                            animate={{
-                                scale: imgIndex === idx ? 0.95 : 0.85,
-                            }}
-                            transition={SPRING_OPTIONS}
-                            className="bg-neutral-800 object-cover h-[15vh] w-[15vh] pt-2"
-                        />
+            {imgs.map((imgSrc, idx) => (                    //come back to this to fix the aspect ratio
+                <div key={idx} className='flex flex-col items-center h-[30vh] w-1/3 rounded-xl bg-neutral-800'>
```

Comment:
Shouldn't use the index as the key

Reply:
changed to items[index].id (the id of the current track using the index to search through items[])